addons/sale_project/models/project.py

```diff
@@ -333,6 +333,12 @@ def _get_profitability_sale_order_items_domain(self, domain=None):
             domain,
         ])
 
+    def _get_materials_action(self, record_ids):
+        args = ['other_revenues', [('id', 'in', record_ids)]]
+        if len(record_ids) == 1:
+            args.append(record_ids[0])
+        return {'name': 'action_profitability_items', 'type': 'object', 'args': json.dumps(args)}
+
```

Comment:
Hello @damrOdoo 👋🏼 
I am so sorry, we forgot that you fixed that issue in this PR. So @xavierbol has made a fix we just merged today.
So your PR will only contain the `allow_billable`.
Can you update your commit message accordingly?

Thanks